src/current/v23.2/alter-database.md

```diff
@@ -834,6 +836,120 @@ The `lease_preferences` field is now updated to include `us-west1`.
 
 To remove the changes made in this example, [reset the Zone Config Extensions](#reset-a-regions-zone-config-extensions).
 
+#### Implement super regions
+
+In this example, we use [Zone Config Extensions]({% link {{ page.version.version }}/zone-config-extensions.md %}) to provide an alternative implementation of [Super Regions]({% link {{ page.version.version }}/multiregion-overview.md %}#super-regions), which are useful for [data domiciling]({% link {{ page.version.version }}/data-domiciling.md %}).
```

Comment:
Per style (feel free to replace my suggestion though) https://github.com/cockroachdb/docs/blob/main/StyleGuide.md#style-and-tone

```suggestion
In this example, [Zone Config Extensions]({% link {{ page.version.version }}/zone-config-extensions.md %}) are used to provide an alternative implementation of [Super Regions]({% link {{ page.version.version }}/multiregion-overview.md %}#super-regions), which are useful for [data domiciling]({% link {{ page.version.version }}/data-domiciling.md %}).
```